pvlib/irradiance.py

```diff
@@ -2915,3 +2916,68 @@ def dni(ghi, dhi, zenith, clearsky_dni=None, clearsky_tolerance=1.1,
             (zenith < zenith_threshold_for_zero_dni) &
             (dni > max_dni)] = max_dni
     return dni
+
+
+def component_sum_irradiance(solar_zenith,
```

Comment:
I think I agree with https://github.com/pvlib/pvlib-python/pull/1567#issuecomment-1268963224 and also prefer the name `complete_irradiance` here

Reply:
Updated in https://github.com/pvlib/pvlib-python/pull/1567/commits/aff38d45d7a1310fff501e50163659d4c3e4e92e